examples/service-clients/azure-client/external-controller/.eslintrc.cjs

```diff
@@ -17,5 +17,16 @@ module.exports = {
 		// Incompatible with formatter
 		// TODO: this can be removed once the eslint config is updated to version 5.4.0 or greater.
 		"@typescript-eslint/brace-style": "off",
+
+		"import/no-internal-modules": [
+			"error",
+			{
+				"allow": [
+					"@fluidframework/*/internal{,/**}",
```

Comment:
Does external controller need /internal? Our examples should mostly not use /internal. Where /internal is used (if just a few place), those can be lint disables at the problem point. Remove this if possible.

Reply:
Removed. Added eslint disables for /alpha usage

Reply:
Arg. I hate that GitHub doesn't let you comment on a single line. The comment was ONLY for `"@fluidframework/*/internal{,/**}",`. The other changes were good and correct. Examples SHOULD use /alpha and /beta so the rule update was good. We need to have ruleset update to support examples code.